modules/nf-core/gmmdemux/main.nf

```diff
@@ -0,0 +1,68 @@
+
+process GMMDEMUX {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gmm-demux:0.2.2.3--pyh7cba7a3_0':
+        'biocontainers/gmm-demux:0.2.2.3--pyh7cba7a3_0' }"
+
+    input:
```

Comment:
**suggestion:** The `-e` flag also [requires a file argument](https://conventionalcomments.org/) and should therefore be included as input.

Reply:
Yes, it used to be in the last version of the package that I submitted, but I was asked to leave optional parameters out. I'll re-add it